src/shared/components/creditBox/CreditBox.tsx

```
@@ -0,0 +1,19 @@
+
+interface CreditBoxProps {
+  credit: number;
+}
+const CreditBox = ({ credit }: CreditBoxProps) => {
+  return (
+    <div className={styles.boxWrapper}>
+      <div className={styles.textContainer}>
+        <span className={styles.infoText}>보유 크레딧</span>
+        <span className={styles.creditText}>{credit}</span>
+      </div>
+      <ChargeButton children="충전하기" />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)